README.md

```diff
@@ -0,0 +1,16 @@
+# **Stock Market**
+
+## Project summary
+In this project I created a system which simulates a stock market. The stock market composed of some companies. The system enables the user to insert and remove companies from the stock market and to make different actions.
+
+##  About the Classes
+Two classes created to implement the system idea:
+
+1. *Company* - This class includes the required methods and represents a specific company.
+
+2. *StockMarket* - This class represents the whole stock market which contains a list of companies.
+
```

Comment:
A better structure here would be:

# project design
## Company class
The company class is described using the following fields: number of stocks, stock price....
The company class api includes the following functions: ....
*you can also add a general explanation about the class and show examples! **Show examples from your tests!**
## StockMarket class
text....

Reply:
done!